README.md

```diff
@@ -40,8 +48,8 @@ npm install moviedb-promise --save
 Require the module and instantiate the class with your themoviedb.org api key.
 
 ```js
-const { MovieDb } = require('moviedb-promise')
-const moviedb = new MovieDb('your api key')
+const { MovieDb } = require("moviedb-promise");
+const moviedb = new MovieDb("your api key");
 ```
```

Comment:
You mentioned it below, but I'm not a semi-colon person, so you can go ahead and drop the semi colons on all the examples. I also prefer single quotes over double. I will add a formatting config, it's overdue.

Reply:
This is another thing prettier decided to do on its own. I'll fix it.